study.md

```diff
@@ -22,7 +22,8 @@ In your own words, explain one reason why we use "this". In your answer, be
 sure to cite any relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+This allows for us to pass values from object to object and to use anonymous
+functions and constructor functions.
```

Comment:
You are close here but `this` is going to refer to the object that a method is being called on.